<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="delto">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.292" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="finger1_link1">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.048" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger1_joint1"
    type="revolute">
    <origin
      xyz="0.0150000000430074 0 0.0829999999999999"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="finger1_link1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-0.804"
      upper="0.804"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger1_link2">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.012" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger1_joint2"
    type="revolute">
    <origin
      xyz="0.0114 -5.8419E-05 0.0091"
      rpy="1.5708 0 1.5657" />
    <parent
      link="finger1_link1" />
    <child
      link="finger1_link2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.714"
      upper="1.714"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger1_link3">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.038" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger1_joint3"
    type="revolute">
    <origin
      xyz="-0.014938 0.0098942 0.0203"
      rpy="1.5708 -1.5707 -1.557" />
    <parent
      link="finger1_link2" />
    <child
      link="finger1_link3" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="2.199"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger1_link4">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.06" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger1_link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger1_joint4"
    type="revolute">
    <origin
      xyz="0.043312 -0.00012363 0"
      rpy="0 0 0" />
    <parent
      link="finger1_link3" />
    <child
      link="finger1_link4" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-0.376"
      upper="2.225"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger2_link1">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.048" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger2_joint1"
    type="revolute">
    <origin
      xyz="-0.0132 0.0196 0.083"
      rpy="0 0 3.1416" />
    <parent
      link="base_link" />
    <child
      link="finger2_link1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-2.147"
      upper="0.788"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger2_link2">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.012" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger2_joint2"
    type="revolute">
    <origin
      xyz="0.014198 -0.00024724 0.0091"
      rpy="0.017412 -1.5708 3.14" />
    <parent
      link="finger2_link1" />
    <child
      link="finger2_link2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.714"
      upper="1.714"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger2_link3">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.038" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger2_joint3"
    type="revolute">
    <origin
      xyz="0.010098 0.014801 0.0175"
      rpy="-1.5708 1.5708 0.00011039" />
    <parent
      link="finger2_link2" />
    <child
      link="finger2_link3" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="2.199"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger2_link4">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.06" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger2_link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger2_joint4"
    type="revolute">
    <origin
      xyz="-0.043282 0.0016118 0"
      rpy="0 0 0" />
    <parent
      link="finger2_link3" />
    <child
      link="finger2_link4" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-0.376"
      upper="2.225"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger3_link1">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.048" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger3_joint1"
    type="revolute">
    <origin
      xyz="-0.0131999999939963 -0.019600000000007 0.0829999999999988"
      rpy="0 0 3.14159265358979" />
    <parent
      link="base_link" />
    <child
      link="finger3_link1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-0.788"
      upper="2.147"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger3_link2">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.012" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger3_joint2"
    type="revolute">
    <origin
      xyz="0.0141973935884402 -0.000272057521322411 0.00910000000000048"
      rpy="3.12243250091804 -1.57079631189374 0" />
    <parent
      link="finger3_link1" />
    <child
      link="finger3_link2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-1.714"
      upper="1.714"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger3_link3">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.038" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger3_joint3"
    type="revolute">
    <origin
      xyz="0.01025 0.014696 0.0175"
      rpy="1.5708 1.5706 3.1314" />
    <parent
      link="finger3_link2" />
    <child
      link="finger3_link3" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="2.199"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
  <link
    name="finger3_link4">
    <inertial>
      <origin
        xyz="0.0163977810461998 -0.000424816867342437 0.078525694814142"
        rpy="0 0 0" />
      <mass
        value="0.06" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://delto/meshes/finger3_link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="finger3_joint4"
    type="revolute">
    <origin
      xyz="-0.043308 -0.0006013 0"
      rpy="0 0 0" />
    <parent
      link="finger3_link3" />
    <child
      link="finger3_link4" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="-0.376"
      upper="2.225"
      effort="10"
      velocity="100" />
    <dynamics damping="0.64" friction="1"/>
  </joint>
</robot>